app/renderer/js/utils/reconnect-util.ts

```diff
@@ -11,11 +11,8 @@ const logger = new Logger({
 	timestamp: true
 });
 
-class ReconnectUtil {
-	// TODO: TypeScript - Figure out how to annotate webview
-	// it should be WebView; maybe make it a generic so we can
-	// pass the class from main.ts
-	webview: any;
+class ReconnectUtil<ClassName> {
+	webview: ClassName ;
```

Comment:
This generic doesn’t make sense because we do `webview.props.url`. TypeScript doesn’t catch that only because you forgot to change the `webview: any` parameter of the constructor. We should just use `webview: WebView` in both places.

Reply:
Did I get the TODO wrong as you're correct that `webview.props.url` even after changing the constructor to `webview: ClassName`

P.S. It was asked to make it generic

Reply:
You read the TODO correctly, but the TODO did not make sense, and we should do things that make sense. :slightly_smiling_face: